docs/execution-providers/Azure-ExecutionProvider.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Azure
+description: Instructions to infer an ONNX model remotely with an Azure endpoint
+parent: Execution Providers
+nav_order: 12
```

Comment:
Should be 11, and community-maintained and add-execution-provider pages need to be shifted to 12 and 13.

Reply:
Done.